app/controllers/control_panel/members_controller.rb

```diff
@@ -0,0 +1,51 @@
+class ControlPanel::MembersController < ControlPanel::ControlPanelController
+before_action :require_owner, except: :index
+
+  def index
+    @board = find_board
+    @members = @board.members.all.decorate
```

Comment:
Yes becouse only owner of board can delete or add the members but members can take a look who is already member of this board